src/lib/umi/confirmAllTransactions.ts

```
@@ -18,15 +13,16 @@ const confirmAllTransactions = async (
 
-  console.log(`Confirming ${transactions.length} transactions`)
-
   let confirmations: UmiTransactionConfirmationResult[] = []
-  
 
   let index = 0
   for (const transaction of transactions) {
-    console.log(`Confirming transaction ${index}`)
-    if (!transaction?.signature) continue
+    if (!transaction?.signature) {
+      onProgress && onProgress(index, { confirmed: false, error: transaction?.err || 'transaction has no signature' })
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)